src/commands/createStaticWebApp/createStaticWebApp.ts

```diff
@@ -22,6 +22,7 @@ import { setWorkspaceContexts } from './setWorkspaceContexts';
 
 export async function createStaticWebApp(context: IActionContext & Partial<ICreateChildImplContext> & Partial<IStaticWebAppWizardContext>, node?: SubscriptionTreeItem): Promise<StaticWebAppTreeItem> {
     if (!node) {
+        context.telemetry.properties.cancelStep = 'showTreeItemPicker';
```

Comment:
As a part of my funnel telemetry work, I'm looking into standardizing and enforcing `cancelStep` a bit more. If you want this reviewed ASAP let me know, but otherwise it might be easier to wait a few days to leverage that

Reply:
Never mind, gonna take me longer than I thought